tests/TranslatableTraitTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace atk4\core\tests;
+
+class TranslatableTraitTest extends \PHPUnit_Framework_TestCase
+{
+    /** @var TranslatableMock */
+    public $translatableMock;
+
+    public function setUp()
+    {
+        $this->translatableMock = new TranslatableMock();
+    }
+
+    public function testStraight()
+    {
+        $trans = $this->translatableMock->_('string without counter');
+        $this->assertEquals('string without counter translated', $trans);
+    }
+
+    public function testSingular()
+    {
+        $trans = $this->translatableMock->_('string not translated simple');
+        $this->assertEquals('string translated', $trans);
+    }
+
+    public function testSingularNotExists()
+    {
+        $trans = $this->translatableMock->_('string not exists');
+        $this->assertEquals('string not exists', $trans);
+    }
+
+    public function testPlurals0()
+    {
+        $trans = $this->translatableMock->_('string not translated with plurals', 0);
```

Comment:
Please use more meaningful examples. I still don’t understand singular/plural logic. Also what if it uses multiple arguments: (3 record(s) imported. 1 error(s))

Reply:
Change some naming and added multiple arguments, see if is better now